### DISCUSSION TITLE
objects copied with `mc cp` are not visible in the console

### DISCUSSION THREAD

**Discussion by knubie** [minio/minio#20725] [Q&A] [answered]
When I use `mc cp` to copy files (either from a local directory or from an s3 bucket), the files get copied into my minio directory on the file system, but they do not show up in the minio console, nor are they found when querying the API.

I noticed that when I upload a file manually via the console, it is stored on the filesystem as something like
```
foo.jpg/
  xl.meta
  uuid/
    part.1
```
However the files that were copied from `mc cp` just exist in the file system as-is.
```
foo.jpg
```

I have the feeling that I am doing something wrong but I could not find any explanation in the docs, or any other similar report online. I do recall this was working at some point, but when it stopped working or why I cannot tell.

* **Version used**: RELEASE.2024-11-07T00-52-20Z (commit-id=cefc43e4daa4cbb490ef6726ea374e26a93eb85e)
* **mc version**: RELEASE.2024-11-05T11-29-45Z (commit-id=6ac18619cf881074fe6edcc79ab62c9c85da60b9)
* **Server setup and configuration**: Running locally on MacOS
* **Operating System and version**: Darwin MacBook-Pro.local 23.2.0 Darwin Kernel Version 23.2.0: Wed Nov 15 21:53:18 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6000 arm64

**Comment by klauspost** [comment, marked as answer]
You must set up your server as an alias in mc.

**Reply by klauspost** [reply]
https://min.io/docs/minio/linux/reference/minio-mc.html#create-an-alias-for-the-s3-compatible-service

**Reply by knubie** [reply]
Hey @klauspost

Thank you for the reply. Your comment made me realize that, while I did have an alias set up, I was mistyping it in the `mc cp` command 😓 

Thank you again so much for your help!
